drive/client.bal

```diff
@@ -69,7 +69,7 @@ public client class Client {
     # 
     # + optional - 'ListFilesOptional' used to add query parameters to the request
     # + return - If successful, returns stream of files `stream<File>`. Else returns `error`
-    @display {label: "Get files"}
+    @display {label: "Get Files"}
     remote isolated function getFiles(@display {label: "Optional Query Parameters"} ListFilesOptional? optional = ()) 
```

Comment:
Shall we name this as `Filtering Parameters`?

Reply:
Now merge all the functionality to `getAllfiles`. Removed this function.